README_ZH.md

```
@@ -59,55 +61,149 @@ export default defineConfig({
+        ifLog: true,
+        ifGlobal: true,
+      },
+    ],
   ],
-})
+});
 ```
 
 至此，你就在开始使用 `vite-plugin-version-mark` 啦! 🎉
 
+> 🚀 **v0.2.0 新功能**：多命令支持！现在你可以执行多个 git 命令并使用自定义格式组合它们的结果。非常适合创建像 `"main-abc1234"` 或 `"v1.0.0-main-abc1234"` 这样的版本字符串。[查看下方示例](#多命令支持--v020-新功能) ⬇️
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)